storage/doc.go

```diff
@@ -224,14 +224,34 @@ BucketHandle.SignedURL method.
 	fmt.Println(url)
 
 You can also sign a URL wihout creating a client. See the documentation of
-SignedURL for details.
+[SignedURL] for details.
 
 	url, err := storage.SignedURL(bucketName, "shared-object", opts)
 	if err != nil {
 	    // TODO: Handle error.
 	}
 	fmt.Println(url)
 
+# Credential detection for [BucketHandle.SignedURL] and [BucketHandle.GenerateSignedPostPolicyV4]
```

Comment:
Maybe this should go underneath the Post Policy section? Also maybe "credential requirements" rather than detection.